### PR TITLE
Appended release notes, prepping for v1.1

### PR DESCRIPTION
**Pull request by jibola1997**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 ﻿RELEASE NOTES
 
+**Version**: 1.1
+
+This versions requires:
+* Shotgun for Unity 1.1 (com.unity.integrations.shotgun)
```
Comment (BadSingleton) [inline RELEASE_NOTES.md:6]: Is this the correct version? usually unity packages versions are like x.y.z[-(pre|exp|rc).w]

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+
+NEW FEATURES
+* Direct Unity project launch(no need for Unity Hub)
```
Comment (BadSingleton) [inline RELEASE_NOTES.md:9]: Space before the opening parenthesis.

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,5 +1,16 @@
...
+* Now working in-process
```
Comment (BadSingleton) [inline RELEASE_NOTES.md:10]: I'm not sure if we need to mention this one here. Or mention that "we updated the API to the 4.0.0 version of the Python for Unity package"

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,5 +1,16 @@
...
+* One-click install for Mac/Win
```
Comment (BadSingleton) [inline RELEASE_NOTES.md:11]: I don't think those 2 needs to be mentionned here; that's more com.unity.integrations.shotgun functionality.

**Review by BadSingleton — CHANGES_REQUESTED** [review]

**Review by lassond — COMMENTED** [review]
Approving, but please do not merge until you sort out the version number.
Why 1.1? 

Back when we stopped working on the Shotgun for Unity package, we decided that the next release (with Python 3 support) would be 1.0

so tk-unity v1.0 and com.unity.integrations.shotgun-1.0.0-preview.1 (not sure about the rules for Unity package version numbers these days)

**Comment by lassond** [comment]
Just for posterity: there is confusion on the version number (v1.0 vs v1.1)
It seems we had started the release process for v1.0 but never completed it

v1.1 is fine

**Comment by jibola1997** [comment]
v1.1 is fine

**Pull request closed without merging**